app/views/shared/_login.html.erb

```diff
@@ -1,4 +1,5 @@
 <!-- ko if: window.tokenExpiresIn === null -->
+<a href="#" data-bind="click: function(){ $root.ui.sitetour.startTour(); }, visible:  $root.ui.sitetour.safePath();" class="toolbar-button">Show Tour</a>
```

Comment:
By the ticket's requirements, a 'Show Tour' needs to be included in the 'Manage Account' drop down, and this is where that drop down is defined, so - this is where the code went.  ;)

The visibility option will hide the button if you are on any page other than '/' or '/search'.